docs/jsonapi-spec.md

```diff
@@ -464,6 +465,42 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
  
 If an error occurs the command will not return `status`.
 
+
+### createCollection Command
+
+Creates a new collection in the current namespace.
+
+*Sample*
+
+```json
+{
+  "createCollection": {
+    "name": "purchase"
+  }
+}
+```
+
+#### createCollection Command Options
+
+The `createCollection` command does not support any options.
+
+#### createCollection Multi Document Failure Modes
```

Comment:
What does this mean? What does multi-document mean in this context? (command only tries to create a single collection)

Reply:
Seems to still remain?